sdk/elasticsans/arm-elasticsan/review/arm-elasticsan.api.md

```diff
@@ -174,19 +172,19 @@ export interface ErrorAdditionalInfo {
 }
 
 // @public
-export interface ErrorModel {
-    error?: ErrorResponse;
-}
-
-// @public
-export interface ErrorResponse {
+export interface ErrorDetail {
     readonly additionalInfo?: ErrorAdditionalInfo[];
     readonly code?: string;
-    readonly details?: ErrorResponse[];
+    readonly details?: ErrorDetail[];
     readonly message?: string;
     readonly target?: string;
 }
 
+// @public
+export interface ErrorResponse {
+    error?: ErrorDetail;
```

Comment:
Could you check the breaking?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/24494